.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -15,3 +15,4 @@ automatically close it when this gets merged.
 - [ ] Add new public functions/methods/classes to `doc/api/index.rst` and the base `__init__.py` file for the package.
 - [ ] Write detailed docstrings for all functions/classes/methods. It often helps to design better code if you write the docstrings first.
 - [ ] If adding new functionality, add an example to the docstring, gallery, and/or tutorials.
+- [ ] Add your full name, affiliation, and ORCID (optional) to the `AUTHORS.md` file (if you haven't already) in case you'd like to be listed as an author on the [Zenodo](https://zenodo.org/communities/fatiando) archive of the next release.
```

Comment:
```suggestion
- [ ] Add your full name, affiliation, and [ORCID](https://orcid.org) (optional) to the `AUTHORS.md` file (if you haven't already) in case you'd like to be listed as an author on the [Zenodo](https://zenodo.org/communities/fatiando) archive of the next release.
```